src/App.tsx

```diff
@@ -1,16 +0,0 @@
-import React from "react";
-import CustomCheckbox from "./components/checkbox";
-import "./App.css";
-
-const App: React.FC = () => {
-  const NumOfCheckbox = [0, 1, 2, 3, 4, 5, 6];
```

Comment:
another better way of writing this is `const NumOfCheckbox = [...Array(7).keys()];`